server/keyshare/keyshareserver/postgresdb.go

```diff
@@ -19,10 +19,7 @@ type postgresDB struct {
 	db keyshare.DB
 }
 
-const maxPinTries = 3                  // number of tries allowed on pin before we start with exponential backoff
-const emailTokenValidity = 24          // amount of time user's email validation token is valid (in hours)
-const emailTokenRateLimit = 3          // max number of active tokens per email address within the emailTokenRateLimitDuration
-const emailTokenRateLimitDuration = 60 // amount of time after which tokens become irrelevant for rate limiting (in minutes)
+var conf *Configuration
```

Comment:
Mostly, you don't want to work with global variables if not necessary. This leads to all kinds of ugly situations in unit and integration tests if you'd run tests in parallel. Both tests will read and write to the same global variable, which can lead to race conditions. It's best practice to add this configuration to the state struct, in this case the `type postgresDB struct`.